libmscore/textbase.cpp

```diff
@@ -1248,6 +1257,7 @@ void TextBase::createLayout()
                   else if (c == '\n') {
                         if (rows() <= cursor.row())
                               _layout.append(TextBlock());
+                        _layout[cursor.row()].insertEmptyFragment(&cursor);
```

Comment:
I think a comment is needed here, why an empty fragment is added.

Reply:
Added